Roadmap.md

```diff
@@ -14,7 +14,9 @@
 
 # Future work planning
 - Automated build for MRTK-Unity.
-- Updating landing page for the ReadMe.md to help app developers find content better.
+- Unity packages for each release will be published to the Unity asset store.
+- Unity packages for each feature area like Input/SpatialMapping will also be added to the release notes. More work needs to happen to ensure feature areas don't have cross dependencies.
```

Comment:
When reducing the cross dependencies, can we make sure that no package REQUIRES the BuildAndDeploy folder/package? While handy, it is not a title side component.

Reply:
Pretty sure that's the idea for each of the folders/packages

Reply:
Although I'm unsure if really needing to do seperate packages for each feature area is really a necessity because when importing a Unity package you already get the option to include/exclude specific folders.

![image](https://user-images.githubusercontent.com/13334553/33048068-dfff0484-ce0d-11e7-8f6e-7e12180f27d5.png)

Reply:
1. About separating feature folders, yes they should not have a dependency on BuildAndDeploy folder for sure. They will have a dependency on Utilities though which is like our Common folder.

2. For separate packages, the winning scenario is porting customers. Sometimes they don't want the brainwork of piecemeal importing the big toolkit. Instead they want simpler packages for say, motion controllers, camera rendering, spatial mapping etc. We should address this feedback. Hence I'm supportive of separate packages for different scenarios and not just feature areas. 
I acknowledge that these packages will be difficult to maintain manually and hence we need an automated story asap :)